railties/lib/rails/generators/rails/app/templates/config/cable.yml

```diff
@@ -7,3 +7,7 @@ test:
 production:
   adapter: redis
   url: redis://localhost:6379/1
+  # To avoid channel collisions when multiple use the same Redis instance,
+  # we prefix channel names with the app/environment name.
+  # (https://redis.io/topics/pubsub#database-amp-scoping)
```

Comment:
This info probably better lives in the "how to configure the adapter" documentation. Do we have any of that at the moment? 😅